code/workspaces/infrastructure-api/src/dataaccess/projectsRepository.js

```diff
@@ -16,6 +16,7 @@ async function exists(projectKey) {
 
 async function create(project) {
   // needs to be in array else thinks is spread of projects and not project then options
+  // as passing projects as array, create returns an array
   return Project().create([project], { setDefaultsOnInsert: true });
```

Comment:
Might it be cleaner to unpack the return array here?

```
const projects = Project().create(...);
return projects[0];
```
This way no later consumers will make the same mistake.

Reply:
Sounds good.

Reply:
Agreed and updated!